### PR TITLE
made up 5 puns on the MRT

### PR DESCRIPTION
**Pull request by Paddlepop25**

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -310,5 +310,25 @@
...
+    "pun": "Q: What do Javascript programmers like to order at the coffeeshop?",
+    "punchline": "A: Mocha, Chai and a cup of Expresso"
+  },
+  {
+    "pun": "Q: Which planet in the solar system would a python developer first travel to?",
+    "punchline": "A: Jupyter"
+  },
+  {
+    "pun": "Q: Why did the developer turn down the marriage proposal?",
+    "punchline": "A: It’s a git commitment"
```
Comment (AlexLakatos) [inline puns.json:324]: Loved all of them except this one. Could you remove this one please? Or come up with a better punchline for it?

**Review by AlexLakatos — CHANGES_REQUESTED** [review]
Thanks for the PR! Loved most of it!

File: puns.json (hunk 1 above)
Comment (Paddlepop25) [reply puns.json]: No problem! I'm changed the git pun. 
I'm not doing committing (via VSCode on my laptop) and doing this directly on the browser on another computer. If I'm not doing anything correctly please let me know :)

**Comment by AlexLakatos** [comment]
Thanks! Happy Hacktoberfest!

**Pull request merged**